app/views/education_experiences/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<ul class="breadcrumb">
+  <li class="breadcrumb-item"><%= link_to t(:back, scope: :helpers), education_experiences_path %></li>
```

Comment:
It looks like this class isn't applying any styles:
![image](https://github.com/Codeminer42/Punchclock/assets/30778707/9862e415-36df-46b6-844c-1c8332ec36e7)

If it's not being used for anything else, you can simply remove it:
```suggestion
  <li><%= link_to t(:back, scope: :helpers), education_experiences_path %></li>
```